client/src/app/App.tsx

```diff
@@ -170,7 +170,12 @@ class App extends Component<RouteComponentProps<AppRouteProps> & { config: IConf
                                         <Route
                                             path="/:network/identity-resolver/:did?"
                                             component={(props: RouteComponentProps<IdentityResolverProps>) => (
-                                                <IdentityResolver {...props} />
+                                                <IdentityResolver
+                                                    {...props}
+                                                    isNetworkChrysalis={
```

Comment:
Instead of `isNetworkChrysalis` I would change to `isSupported` for when future protocols are implemented